src/native/corehost/fxr/standalone/hostpolicy_resolver.cpp

```diff
@@ -43,13 +42,13 @@ namespace
             if (utils::starts_with(lib_name, prefix, false))
             {
                 // Extract the version information that occurs after '/'
-                retval = lib_name.substr(utils::strlen(prefix));
-                break;
+                pal::string_t version = lib_name.substr(utils::strlen(prefix));
+                trace::verbose(_X("Resolved version %s from dependency manifest file [%s]"), version.c_str(), deps_json.c_str());
```

Comment:
Not actually necessary for this change, but this trace message (with no version) was confusing me. I actually think we could/should get rid of this entire 'look for the hostpolicy package in the .deps.json' (we no longer ship it and it hasn't been how apps use hostpolicy since before I was around) situation, but that is a separate issue.